src/HttpKernel/ContaoCache.php

```diff
@@ -10,11 +10,44 @@
 
 namespace Contao\ManagerBundle\HttpKernel;
 
+use FOS\HttpCache\SymfonyCache\CacheInvalidation;
+use FOS\HttpCache\SymfonyCache\EventDispatchingHttpCache;
```

Comment:
Not needed, the `header-replay-bundle` requires that and there's never a case where it would not because it only ever works with this.